pkg/asset/manifests/aws/cluster.go

```diff
@@ -151,19 +152,53 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 				PresignedURLDuration: &metav1.Duration{Duration: 1 * time.Hour},
 			},
 			ControlPlaneLoadBalancer: &capa.AWSLoadBalancerSpec{
-				Name:             ptr.To(clusterID.InfraID + "-ext"),
+				Name:             ptr.To(clusterID.InfraID + "-int"),
 				LoadBalancerType: capa.LoadBalancerTypeNLB,
-				Scheme:           &capa.ELBSchemeInternetFacing,
+				Scheme:           &capa.ELBSchemeInternal,
 				AdditionalListeners: []capa.AdditionalListenerSpec{
 					{
 						Port:     22623,
 						Protocol: capa.ELBProtocolTCP,
 					},
 				},
+				IngressRules: []capa.IngressRule{
+					{
+						Description: "MCS traffic from cluster",
+						Protocol:    capa.SecurityGroupProtocolTCP,
+						FromPort:    22623,
+						ToPort:      22623,
+						CidrBlocks:  []string{mainCIDR.String()},
+					},
+					{
+						Description: "Kube API traffic",
+						Protocol:    capa.SecurityGroupProtocolTCP,
+						FromPort:    6443,
+						ToPort:      6443,
+						CidrBlocks:  []string{"0.0.0.0/0"},
+					},
```

Comment:
@r4f4 based in our conversation on Slack, do we need that allow-all by default here, or only when the publish is External?

Reply:
Good point. Will fix.

Reply:
Fixed.